recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "osicbc" %}
+{% set sha256 = "577c9a43506d1e171c80c9b6132ba2ed2a91d4d448c9c1e78d6fb794301536ef" %}
+{% set variant = "openblas" %}
+
+package:
+  name: cyclus
+  version: {{ version }}
+
+source:
+  fn: cyclus-{{ version }}-{{ sha256 }}.tar.gz
+  url: https://github.com/scopatz/cyclus/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Could we change this selector to something like `# [win or not py35]` or similar? Not sure what Python version is best. However, picking one out Python 2.7, 3.4, and 3.5 would help speed this build up and avoid some confusing output from `conda-build-all` (and possibly the feedstock).